packages/node/src/utils.ts

```diff
@@ -24,8 +25,11 @@ function getRegExpFromMatcher(matcher: unknown): string {
     );
   }
 
-  const re = pathToRegexp(matcher);
-  return re.source;
+  const regExps = [pathToRegexp(matcher).source];
+  if (matcher === '/') {
+    regExps.push(pathToRegexp('/index').source);
```

Comment:
**suggestion:** check for `/index` already existing before adding in

If the matchers are `['/', '/index']`, the result is that the regex for `/index` is in the resulting regex twice. I don't think this will cause any problems, but we should probably keep it unique.

Reply:
A good idea Sean.
I'll open a followup PR